src/roles/clusters/openshift/crc/crc-install/config.yaml

```
@@ -0,0 +1,83 @@
+    required: true
+  - name: ADD_CLUSTER_ADMIN_TO_DEVELOPER
+    description: |
+      Set ADD_CLUSTER_ADMIN_TO_DEVELOPER to true if you want to add cluster admin role to developer user (default: false)
+    default: false
+  - name: RESET_CRC_SETUP
+    description: |
+      Set RESET_CRC_SETUP to true if you want to reset crc setup (default: false)
+    default: false
+
+  output_env:
+    - name: CLUSTER_CONSOLE_URL
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)